### PR TITLE
Add info about the max size property

### PR DESCRIPTION
**Pull request by shomix**

### CODE REVIEW

File: pages/sdk/storage-v2.mdx [hunk 1]
```diff
@@ -231,13 +231,15 @@ Used to create a token that can be used to upload data directly from browser, us
...
   protocol,
+  maxSize,
 });
 ```
 
 Function `createSingleUploadToken` takes two parameters:
 
 - **name** - (string) - Represents the name of the bucket on which you are uploading the data.
 - **protocol** - (string) - The protocol on which the data will be uploaded. The supported protocols are [ `ARWEAVE`, `IPFS`, `FILECOIN`].
+- **maxSize** - (number) _optional_ - The maximum size of the upload in bytes. If the uploaded data size is exceeded an error will be thrown.
```
Comment (aayushmahapatra) [inline pages/sdk/storage-v2.mdx:242]:
```suggestion
- **maxSize** - (number) _(optional)_ - The maximum size of the upload in bytes. If the uploaded data size is exceeded an error will be thrown.
```

File: pages/sdk/storage-v2.mdx [hunk 2]
```diff
@@ -266,7 +268,7 @@ Function `pinCID` takes one parameter:
 - **configuration:** An object with the following parameters:
   - `configuration.name` (string) - Represents the name of the bucket on which you are pinning the CID.
   - `configuration.cid` (string) - The CID of the file you want to be pinned
-  - `configuration.inBackground` (string) - **(optional)** - when set to true, the pinning will processed in the background. Using the `uploadId` from the response, the status can be checked via [getUpload](/sdk/storage-v2/#get-upload) function.
+  - `configuration.inBackground` (boolean) - **(optional)** - when set to true, the pinning will processed in the background. Using the `uploadId` from the response, the status can be checked via [getUpload](/sdk/storage-v2/#get-upload) function.
```
Comment (aayushmahapatra) [inline pages/sdk/storage-v2.mdx:271]:
```suggestion
  - `configuration.inBackground` (boolean) - _(optional)_ - when set to true, the pinning will processed in the background. Using the `uploadId` from the response, the status can be checked via [getUpload](/sdk/storage-v2/#get-upload) function.
```

**Review by aayushmahapatra — CHANGES_REQUESTED** [review]
Let's keep the pattern consistent.

File: pages/sdk/storage-v2.mdx (hunk 1 above)
Comment (shomix) [reply pages/sdk/storage-v2.mdx]: dione

File: pages/sdk/storage-v2.mdx (hunk 2 above)
Comment (shomix) [reply pages/sdk/storage-v2.mdx]: done

**Review by aayushmahapatra — APPROVED** [review]

**Pull request merged**